tla-pp/src/test/scala/at/forsyte/apalache/tla/pp/TestKeramelizer.scala

```diff
@@ -298,6 +298,82 @@ class TestKeramelizer extends AnyFunSuite with Checkers with BeforeAndAfterEach
     assert(expected == output)
   }
 
+  test("""A \subseteq B ~~> \A a \in A: a \in B""") {
+    val types = Map("BOOL" -> BoolT1(), "SET" -> SetT1(IntT1()), "INT" -> IntT1())
+    def A = tla.name("A") ? "SET"
+    def B = tla.name("B") ? "SET"
+    val input =
+      tla
+        .subseteq(A, B)
+        .typed(types, "BOOL")
+    val output = keramelizer.apply(input)
+    def t1 = tla.name("t_1") ? "INT"
+    val expected: TlaEx =
+      tla
+        .forall(t1, A, tla.in(t1, B) ? "BOOL")
+        .typed(types, "BOOL")
+    assert(expected == output)
```

Comment:
This is a bit fragile, because you're expecting a specific name from the name generator, which is an implementation detail. 
I'd rather do something like:
```scala
val isExpected = output match {
  case OperEx( TlaBoolOper.forall, NameEx(boundName), `A`, OperEx(TlaSetOper.in, NameEx(inName), `B`) ) => boundName == inName
  case _ => false
}
assert(isExpected)
```

Same for the other tests.